README.md

```diff
@@ -53,7 +53,8 @@ If you get SSL error when running `nix-shell` (SSL peer certificate or SSH remot
 
 1. Run `yarn nix:selfnode` from `daedalus`.
 2. Run `yarn dev` from the subsequent `nix-shell` (use `KEEP_LOCAL_CLUSTER_RUNNING` environment variable to keep the local cluster running after Daedalus exits: `KEEP_LOCAL_CLUSTER_RUNNING=true yarn dev`)
-3. Once Daedalus has started and has gotten past the loading screen run the following commands from a new terminal window if you wish to import funded wallets:
+3. Run `yarn start` from a second `nix-shell` instance (running step 1 from a terminal instance)
```

Comment:
Whats the main usage of `yarn start`? My current workflow is `yarn nix:testnet && yarn dev`, is it any different?

Reply:
`yarn start` can be used after `yarn build` to test a production build